### PR TITLE
build(install): improve root detection and set ownership on files

### LINKED ISSUE #187: bug(install): if project installed by root or folder owned by root celery is not starting
### Current Behavior

Due to bad permissions on the docker volume of the project folder Celery container won't start

### Expected Behavior

All should works well after install

### Steps To Reproduce

Launch install.sh from root user or with project folder having the root permission

### Environment

```markdown
- reNgine: 2.1.0
- OS: Debian 12
- Python: 
- Docker Engine: 
- Docker Compose: 
- Browser:
```

### Anything else?

_No response_

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

### PR DESCRIPTION
**Pull request by psyray**
Fixes #187 

This PR improves installation and resolve problems when user install as root user or if the project folder has root rights

As we move to the rengine user inside docker, root install is not possible.
And files needs to be set to the sudo user installing reNgine-ng to prevent bad permissions inside the docker volumes
So I've added more check for root user and a chown with the current user on the project files
It appears like this if the install change the project folder permissions
image

Tested under root directly -> Install exit
Tested under sudo su -> Install exit
Tested under sudo -> Install continue and reset the project folder permission to the current user if it was not

I've also put some warning for the PG_USER part in the .env-dist file
And I've remove the useless make.bat

### CODE REVIEW

File: .env-dist [hunk 1]
```diff
@@ -15,7 +15,8 @@ STATE=Georgia
 CITY=Atlanta
 
 #
-# Database configurations
+# Database configurations 
+# /!\ POSTGRES_USER & PG_USER must be the same user or Celery will fail to start
```
Comment (AnonymousWP) [inline .env-dist:19]: What are the front and backslash for here actually?

File: install.sh [hunk 2]
```diff
@@ -12,12 +12,32 @@ for ip in $internal_ips; do
 done
 
 # Check for root privileges
-if [ "$(whoami)" != "root" ]
-  then
-  log ""
-  log "Error installing reNgine-ng: please run this script as root!" $COLOR_RED
+if [ $EUID -eq 0 ]; then
+  if [ "$SUDO_USER" = "root" ] || [ "$SUDO_USER" = "" ]; then
+    log "Error: Do not run this script as root. Use sudo with a non-root user." $COLOR_RED
```
Comment (AnonymousWP) [inline install.sh:17]:
```suggestion
    log "Error: Do not run this script as root user. Use `sudo` with a non-root user." $COLOR_RED
```

Backticks should work as it's within `log`. And is it worth mentioning that the non-root user should be in `sudoers`?

File: install.sh [hunk 3]
```diff
@@ -12,12 +12,32 @@ for ip in $internal_ips; do
...
+    log "Example: sudo ./install.sh" $COLOR_RED
```
Comment (AnonymousWP) [inline install.sh:18]:
```suggestion
    log "Example: `sudo ./install.sh`" $COLOR_RED
```

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

File: .env-dist (hunk 1 above)
Comment (psyray) [reply .env-dist:19]: To make a warning ⚠️ panel 😉

File: install.sh (hunk 2 above)
Comment (psyray) [reply install.sh:17]: Not sure but why do not put simple quote I stead of back tick ?

File: install.sh (hunk 3 above)
Comment (psyray) [reply install.sh:18]: Same comment

File: install.sh (hunk 2 above)
Comment (psyray) [reply install.sh]: Replaced by single quote

File: install.sh (hunk 3 above)
Comment (psyray) [reply install.sh]: Replaced by single quote

**Review by AnonymousWP — APPROVED** [review]

**Pull request merged**